sql/core/src/test/resources/sql-tests/inputs/postgreSQL/float4.sql

```diff
@@ -106,7 +106,7 @@ SELECT smallint(float('32767.6'));
 SELECT smallint(float('-32768.4'));
 SELECT smallint(float('-32768.6'));
 SELECT int(float('2147483520'));
-SELECT int(float('2147483647'));
+SELECT int(float('2147483392'));
```

Comment:
let's NOT change the pgsql tests. They are used to verify the difference between Spark and pgsql. We should respect the test result, whatever it is.